### PR TITLE
fix:spire-quickstart.yaml path is incorrect in doc "Install SPIRE Option 1: Quick start" #11374

### PR DESCRIPTION
**Pull request by yanrongshi**
Please provide a description for what this PR is for.
fix: #11374 
And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/ops/integrations/spire/index.md [hunk 1]
```diff
@@ -226,7 +226,7 @@ To improve workload attestation security robustness, SPIRE is able to verify aga
 1. Deploy an example workload:
 
     {{< text bash >}}
-    $ istioctl kube-inject --filename @samples/security/spire/sleep-spire.yaml | kubectl apply -f -
+    $ istioctl kube-inject --filename samples/security/spire/sleep-spire.yaml | kubectl apply -f -
```
Comment (rvennam) [inline content/en/docs/ops/integrations/spire/index.md:229]: Have you tried `@samples/security/spire/sleep-spire.yaml@` ?
See: https://github.com/istio/istio.io/blob/master/content/en/docs/setup/additional-setup/sidecar-injection/index.md

Comment (frankbu) [reply content/en/docs/ops/integrations/spire/index.md:229]: `@samples/security/spire/sleep-spire.yaml@` will make it a clickable link in the rendered doc that will point to the file in the istio/istio repo (https://github.com/istio/istio/blob/master/samples/security/spire/spire-quickstart.yaml). That would be a good feature to use here, as @rvennam suggested. The two `@` chars are stripped when you copy/paste the command.

Comment (yanrongshi) [reply content/en/docs/ops/integrations/spire/index.md:229]: Close the issue #11374 and keep the source document？

Comment (frankbu) [reply content/en/docs/ops/integrations/spire/index.md:229]: @yanrongshi No, the fix for issue #11374 is to add a second `@` char at the end, not remove the first one.
```suggestion
    $ istioctl kube-inject --filename @samples/security/spire/sleep-spire.yaml@ | kubectl apply -f -
```

**Review by craigbox — APPROVED** [review]

**Pull request merged**